srtcore/core.cpp

```diff
@@ -4757,33 +4759,22 @@ void *CUDT::tsbpd(void *param)
                      * packet ready to play but preceeded by missing packets (hole).
                      */
 
-                    /* Update drop/skip stats */
-                    enterCS(self->m_StatsLock);
-                    self->m_stats.rcvDropTotal += seqlen;
-                    self->m_stats.traceRcvDrop += seqlen;
-                    /* Estimate dropped/skipped bytes from average payload */
-                    int avgpayloadsz = self->m_pRcvBuffer->getRcvAvgPayloadSize();
-                    self->m_stats.rcvBytesDropTotal += seqlen * avgpayloadsz;
-                    self->m_stats.traceRcvBytesDrop += seqlen * avgpayloadsz;
-                    leaveCS(self->m_StatsLock);
-
-                    self->dropFromLossLists(self->m_iRcvLastSkipAck,
-                                            CSeqNo::decseq(skiptoseqno)); // remove(from,to-inclusive)
+                    self->updateForgotten(seqlen, self->m_iRcvLastSkipAck, skiptoseqno);
                     self->m_pRcvBuffer->skipData(seqlen);
 
                     self->m_iRcvLastSkipAck = skiptoseqno;
 
 #if ENABLE_LOGGING
                     int64_t timediff_us = 0;
                     if (!is_zero(tsbpdtime))
-                        timediff_us = count_microseconds(tsbpdtime - steady_clock::now());
+                        timediff_us = count_microseconds(steady_clock::now() - tsbpdtime);
```

Comment:
Debug value fix. This was displayed usually when `tsbpdtime` was already in the past, so it was practically always displaying a negative value.